iFixit/react-commerce
# Add catch all route for page template

closes #1594 

## QA

1. Open the [Strapi Govinor instance for this branch](https://add-page-template-catchall-route.govinor.com/admin/content-manager/collectionType/api::page.page?page=1&pageSize=10&sort=path:ASC&plugins[i18n][locale]=en)
2. Go to `Content Manager` > `Page` and create a new page
3. Set the page path to `/test` and add as many sections as you wish
4. Publish the page
5. Open the [preview page](https://react-commerce-git-add-page-template-catchall-route-ifixit.vercel.app/test) (the link points to `/test`) and verify that it shows the page you've just created
